packages/vuetify/src/composables/display.ts

```diff
@@ -0,0 +1,198 @@
+// Utilities
+import { computed, inject, ref } from 'vue'
+
+// Globals
+import { IN_BROWSER, SUPPORTS_INTERSECTION, SUPPORTS_TOUCH } from '@/util/globals'
+
+// Types
+import type { InjectionKey, Ref } from 'vue'
+import { mergeDeep } from '@/util'
+
+export type DisplayBreakpoint = keyof DisplayThresholds
+
+export interface DisplayThresholds {
+  xs: number
+  sm: number
+  md: number
+  lg: number
+  xl: number
+}
+
+export interface DisplayOptions {
+  mobileBreakpoint?: number | DisplayBreakpoint
+  scrollBarWidth?: number
+  thresholds?: Partial<DisplayThresholds>
+}
+
+export interface InternalDisplayOptions {
+  mobileBreakpoint: number | DisplayBreakpoint
+  scrollBarWidth: number
+  thresholds: DisplayThresholds
+}
+
+export interface DisplayInstance {
+  display: Ref<{
+    height: number
+    lg: boolean
+    lgAndDown: boolean
+    lgAndUp: boolean
+    lgOnly: boolean
+    md: boolean
+    mdAndDown: boolean
+    mdAndUp: boolean
+    mdOnly: boolean
+    name: DisplayBreakpoint
+    sm: boolean
+    smAndDown: boolean
+    smAndUp: boolean
+    smOnly: boolean
+    xl: boolean
+    xlOnly: boolean
+    xs: boolean
+    xsOnly: boolean
+    mobile: boolean
+    mobileBreakpoint: number | DisplayBreakpoint
+    thresholds: DisplayThresholds
+    scrollBarWidth: number
+    width: number
+  }>
+}
+
+export const VuetifyDisplaySymbol: InjectionKey<DisplayInstance> = Symbol.for('vuetify:display')
+
+const defaultDisplayOptions: DisplayOptions = {
+  mobileBreakpoint: 'md',
+  scrollBarWidth: 16,
+  thresholds: {
+    xs: 600,
+    sm: 960,
+    md: 1280,
+    lg: 1920,
+    xl: 3840,
```

Comment:
isn't it redundant? `xl` would be just anything that's greater than `lg` (1920)

Reply:
for future expansion, we can remove if needed